dbms/src/DataStreams/IBlockInputStream.h

```diff
@@ -212,6 +212,9 @@ class IBlockInputStream
 
         /// in rows per second
         size_t min_execution_speed = 0;
+        size_t max_execution_speed = 0;
+        size_t min_execution_bytes_speed = 0;
```

Comment:
min/max_execution_bytes_speed
-> min/max_execution_speed_bytes

to keep it uniform with other settings.